packages/paste-core/components/button/src/index.tsx

```diff
@@ -108,38 +116,20 @@ const Button: React.FC<ButtonProps> = props => {
 
 Button.defaultProps = {
   as: 'button',
-  type: 'button',
```

Comment:
We may want to keep the default type as button still

Reply:
Added back `type` prop.